mono/metadata/sysmath.c

```diff
@@ -316,3 +316,142 @@ ves_icall_System_Math_SplitFractionDouble (gdouble *v)
 {
 	return modf (*v, v);
 }
+
+gfloat
+ves_icall_System_MathF_Acos (gfloat x)
```

Comment:
This should be just float, we don't use gfloat.

Reply:
We do in that file, just a couple of lines above ;-)

I dont mind updating.